src/systemTest/java/com/flightstats/hub/model/ChannelItemPathParts.java

```diff
@@ -0,0 +1,24 @@
+package com.flightstats.hub.model;
+
+import lombok.Builder;
+import lombok.Getter;
+import org.joda.time.DateTime;
+
+@Getter
+@Builder
+public class ChannelItemPathParts {
```

Comment:
It does feel that way. I'd rather make a common library of dumb-ish domain objects that we can use in main and test so that these system tests don't have a dependency on the main package. I tend to like system tests to be black box. If we split it out into a library, other teams could use it if they didn't feel like maintaining their own hub library, too...

I think that's outside the scope of this PR, but I'd definitely like to do cleanup like that.